source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -2371,3 +2372,40 @@ message TransformationWrapRegionInSelection {
   bool branch_condition = 3;
 
 }
+
+message TransformationWrapVectorSynonym {
+  // A transformation that wraps an arithmetic operation into a vector operation
+  // and get the result of the original operation from the corresponding index.
+  // For instance, after the transformation, an addition operation between two scalars:
+  // c = a + b
+  //
+  // will be wrapped into vectors:
+  //
+  // va = vector(..., a, ...)
+  // vb = vector(..., b, ...)
+  // vc = va + vb
+  //
+  // where the result will be updated from the corresponding index of the result vector:
+  //
+  // c = vc[corresponding_idx]
+
+  // The result if of the original scalar operation instruction.
+  uint32 instruction_id = 1;
+
+  // The result id for the first vector that contains the first value of the scalar operation.
+  uint32 result_id1 = 2;
+
+  // The result id for the second vector that contains the second value of the scalar operation.
+  uint32 result_id2 = 3;
```

Comment:
I suggest "vector_operand1" and "vector_operand2" as names here.

Reply:
Done.